Majasdarbs/06-majasdarbs/OlegsKozlovs/app.js

```diff
@@ -0,0 +1,30 @@
+const characters = ["a", "b", "c", "d", "e", "g", "k", "m", "o", "q", "t", "z"];
+
+let arrayId;
+
+let output = document.getElementById("main");
+
+const printValues = () => { output.innerText = characters[arrayId]};
```

Comment:
Also  idea: to make this more flexible you could pass in here either an output element, so it is dynamic